integration-tests/src/test/java/oracle/kubernetes/operator/utils/JrfDomain.java

```diff
@@ -61,7 +61,8 @@ private void updateDomainMapForJrf(boolean adminPortEnabled) throws Exception {
     if (System.getenv("IMAGE_PULL_SECRET_FMWINFRA") != null) {
       domainMap.put("imagePullSecretName", System.getenv("IMAGE_PULL_SECRET_FMWINFRA"));
     } else {
-      domainMap.put("imagePullSecretName", "ocir-store");
+      //domainMap.put("imagePullSecretName", "ocir-store");
```

Comment:
can you remove this commented out code please

Reply:
Done